lib/rules/declaration-property-value-keyword-no-deprecated/__tests__/index.mjs

```diff
@@ -81,6 +90,30 @@ testRule({
 			endLine: 1,
 			endColumn: 41,
 		},
+		{
+			code: 'a { overflow: overlay; }',
+			message: messages.rejected('overflow', 'overlay'),
+			line: 1,
+			column: 15,
+			endLine: 1,
+			endColumn: 22,
+		},
+		{
+			code: 'a { overflow: hidden overlay; }',
+			message: messages.rejected('overflow', 'overlay'),
+			line: 1,
+			column: 22,
+			endLine: 1,
+			endColumn: 29,
+		},
+		{
+			code: 'a { overflow: hidden /* comment */ overlay; }',
+			message: messages.rejected('overflow', 'overlay'),
+			line: 1,
+			column: 36,
+			endLine: 1,
+			endColumn: 43,
+		},
```

Comment:
Let's move these test cases to the previous `testRule` block, as they are unrelated to the testing the secondary option. If they're unfixable, you can use [`unfixable: true`](https://github.com/stylelint/jest-preset-stylelint/blob/0b550662b23ad1449e5fdb7227c175cf6f46ce86/index.d.ts#L76).

Let's also open an issue (or do a quick follow-up PR) to add `fix: true` to this `testRule` block, as we typically do that in every block for rules that support autofix.

Reply:
> Let's also open an issue (or do a quick follow-up PR) to add fix: true to this testRule block, as we typically do that in every block for rules that support autofix.

Ill do that in the PR that will add support for `border-color` and `scrollbar-color`.
This PR needs to be merged first.